drupal/usr/local/share/drupal/drupal_functions.sh

```diff
@@ -90,20 +90,104 @@ do_drupal_install() {
   # Drop the database if we need to force the install every time.
   if [ "${FORCE_DATABASE_DROP}" == 'true' ]; then
     echo 'Dropping the Drupal DB if it exists'
-    as_code_owner "drush sql-drop -y -r ${WEB_DIRECTORY}"
+    as_code_owner "drush ${DRUPAL_DRUSH_ALIAS} sql-drop -y -r ${WEB_DIRECTORY}"
   fi
 
   # If we're supposed to install Drupal, and it's not currently installed,
   # then install it.
   if ! drush status bootstrap -r "${WEB_DIRECTORY}" | grep -q Successful ; then
     echo 'Installing Drupal'
-
+    set +x
     INSTALL_OPTS="${DRUPAL_INSTALL_PROFILE} --account-name=\"${DRUPAL_ADMIN_USERNAME}\"  --account-pass=\"${DRUPAL_ADMIN_PASSWORD}\" install_configure_form.update_status_module='array(FALSE,FALSE)'"
     # We should make sure this is writeable, but only do it directly before an
     # install. Drupal will lock it back down on install completion.
     chmod -R ug+rw,o-w "${WEB_DIRECTORY}/sites/default/files"
     chmod go+w "${WEB_DIRECTORY}/sites/default/settings.php"
-    as_code_owner "drush site-install ${INSTALL_OPTS} -y -r ${WEB_DIRECTORY}"
+    as_code_owner "drush ${DRUPAL_DRUSH_ALIAS} site-install ${INSTALL_OPTS} -y -r ${WEB_DIRECTORY}"
```

Comment:
As above, do not use alias here as there is a risk of destroying data elsewhere.